lib/streamlit/elements/exception_proto.py

```diff
@@ -118,7 +145,32 @@ def marshall(exception_proto, exception, exception_traceback=None):
         )
 
 
-def get_stack_trace(exception, exception_traceback=None):
+def get_stack_trace(
```

Comment:
Make this private (i.e. prefix with underscore). Same everywhere on this file where this applies — I haven't checked :wink: